doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -0,0 +1,373 @@
+# The Node.js Event Loop, Timers, and `process.nextTick()`
+
+## What is the Event Loop?
+
+The event loop is what allows Node.js to perform non-blocking I/O
+operations — despite the fact that JavaScript is single-threaded — by
+offloading operations to the system kernel whenever possible. 
+
+Since most modern kernels are multi-threaded, they can handle
+multiple operations executing in the background. When one of these
+operations completes, the kernel tells Node.js so that the appropriate callback 
+may added to the `poll` queue to eventually be executed. We'll explain
+this in further detail later in this topic. 
+
+## Event Loop Explained 
+
+When Node.js starts, it initializes the event loop, processes the provided
+input script (or drops into the REPL, which is not covered in this document)
+which may make async API calls, schedule timers, or call `process.nextTick()`, 
+then begins processing the event loop. 
+
+The following diagram shows a simplified overview of the event loop's
+order of operations.
+
+       ┌───────────────────────┐
```

Comment:
@mcollina It's because we [run `uv_run()` using `UV_RUN_ONCE`](https://github.com/nodejs/node/blob/v5.9.1/src/node.cc#L4151). Which we started using [for `'beforeExit'` support](https://github.com/nodejs/node/commit/a2eeb43). The reason libuv does this is [documented in `deps/uv/src/unix/core.c`](https://github.com/nodejs/node/blob/v5.9.1/deps/uv/src/unix/core.c#L355-L365)